### ISSUE TITLE
Difficult to identify which field the word “required” belongs to in the forms

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#3067]
Originally from:
- https://github.com/GCTC-NTGC/gc-accessibility-passport/issues/118

### Description
From AAACT:

> It is not easy to see which field the word “required” belongs to in the forms. They should be in close proximity to their associated fields.
> Although in this specific case all fields are required and the user cannot misunderstand anyway, right aligning them may become confusing in forms with some fields that are not required.

Screenshot
<img width="377" alt="68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3566376233333337336163376237323739646539613363352f30303631333333322d396237612d343033372d613134352d393661653966383762623732" src="https://user-images.githubusercontent.com/3046459/173676465-5e4817d1-00b3-49ea-8871-0eedd866c35c.png">

**Comment by mnigh** [comment]
> @mnigh Was re-opening this issue an accident or did the attached PR miss something?

Yes, whoops.
